kore/test/Test/Kore/Unification/Unifier.hs

```diff
@@ -655,11 +652,9 @@ test_unsupportedConstructs =
         andSimplifyFailure
```

Comment:
Is there a difference now between `andSimplifySuccess` and `andSimplifyFailure`?

Reply:
You're right, there isn't any. I've made the necessary changes. Thanks!